server.js

```diff
@@ -0,0 +1,23 @@
+const express = require("express");
+const server = express();
+
+server.use(express.json());
+server.get("/",(req,res) => res.send("Helloooooooooooooo"));
+
+server.post("/games", (req,res) => {
+	const { title, genre } = req.body;
+	if(title && genre){
+		res.status(200).json(req.body);
+	} else { 
+		res
+			.status(422)
+			.json({message : "Missing Info"});
```

Comment:
A more help message would be something like 
```suggestion
			.json({message : "Game is missing title and/or genre."});
```